src/main/java/ru/jru/quest/karaganov/logic/Game.java

```diff
@@ -0,0 +1,38 @@
+package ru.jru.quest.karaganov.logic;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.Getter;
+import lombok.SneakyThrows;
+import ru.jru.quest.karaganov.Repository;
+import ru.jru.quest.karaganov.entitie.Step;
+import java.io.File;
+import java.util.*;
+
+
+@Getter
+public class Game {
+    private final ArrayList<Step> steps;
+    private final Repository repository;
+
+    public Game(){
+        repository = new Repository();
```

Comment:
Can be constructor injection?

Reply:
Repository() and ObjectMapper() constuctor injections are added